### ISSUE TITLE
FileConverter

### ISSUE THREAD

**Issue by stereo720712** [ONLYOFFICE/build_tools#97]
when build finish
i follow the step 🥇 
Start the FileConverter service:
```
cd out/linux_64/onlyoffice/documentserver/server/FileConverter
LD_LIBRARY_PATH=$PWD/bin NODE_ENV=development-linux NODE_CONFIG_DIR=$PWD/../Common/config ./converter
```
and then shows the error:

```
root@ubuntu:~/doc_ser/build_tools-hotfix-v5.5.3/out/linux_64/onlyoffice/documentserver/server/FileConverter# ./converter 
WARNING: No configurations found in configuration directory:/root/doc_ser/build_tools-hotfix-v5.5.3/out/linux_64/onlyoffice/documentserver/server/FileConverter/config
WARNING: To disable this warning set SUPPRESS_NO_CONFIG_WARNING in the environment.
pkg/prelude/bootstrap.js:1244
      throw error;
      ^

Error: Configuration property "log.filePath" is not defined
    at Config.get (/snapshot/server/build/server/Common/node_modules/config/lib/config.js:203:11)
    at Object.<anonymous> (/snapshot/server/build/server/Common/sources/logger.js:0:0)
    at Module._compile (pkg/prelude/bootstrap.js:1320:22)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:651:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:591:12)
    at Function.Module._load (internal/modules/cjs/loader.js:583:3)
    at Module.require (internal/modules/cjs/loader.js:690:17)
    at Module.require (pkg/prelude/bootstrap.js:1225:31)
    at require (internal/modules/cjs/helpers.js:25:18)
root@ubuntu:~/doc_ser/build_tools-hotfix-v5.5.3/out/linux_64/onlyoffice/documentserver/server/FileConverter# A

======
```
can anyone help , thanks a lot .

**Comment by stereo720712** [comment]
and DocService is start fail too with the follow message: root@ubuntu:~/doc_ser/build_tools-hotfix-v5.5.3/out/linux_64/onlyoffice/documentserver/server/DocService# ./docservice 
WARNING: No configurations found in configuration directory:/root/doc_ser/build_tools-hotfix-v5.5.3/out/linux_64/onlyoffice/documentserver/server/DocService/config
WARNING: To disable this warning set SUPPRESS_NO_CONFIG_WARNING in the environment.
/snapshot/server/build/server/DocService/node_modules/config/lib/config.js:203
    throw new Error('Configuration property "' + property + '" is not defined');
    ^

Error: Configuration property "services.CoAuthoring" is not defined
    at Config.get (/snapshot/server/build/server/DocService/node_modules/config/lib/config.js:203:11)
    at Object.<anonymous> (/snapshot/server/build/server/DocService/sources/server.js:0:0)
    at Module._compile (pkg/prelude/bootstrap.js:1320:22)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:651:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:591:12)
    at Function.Module._load (internal/modules/cjs/loader.js:583:3)
    at Function.Module.runMain (pkg/prelude/bootstrap.js:1375:12)
    at startup (internal/bootstrap/node.js:320:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:660:3)
root@ubuntu:~/doc_ser/build_tools-hotfix-v5.5.3/out/linux_64/onlyoffice/documentserver/server/DocService#

**Comment by ShockwaveNN** [comment]
Hi, that is content of `out/linux_64/onlyoffice/documentserver/server/Common/config/`
Should be:
```
tree out/linux_64/onlyoffice/documentserver/server/Common/config/
out/linux_64/onlyoffice/documentserver/server/Common/config/
├── default.json
├── development-linux.json
├── development-mac.json
├── development-windows.json
├── log4js
│   ├── development.json
│   └── production.json
├── production-linux.json
└── production-windows.json

1 directory, 8 files
```

Also, on unrated note - your build use `build_tools-hotfix-v5.5.3` name.
My advice - if you build from something other than master - it can be broken on any level. So my advice build only from master until you 100% what are you doing

**Comment by stereo720712** [comment]
thanks a lot , Pavel , Because i fail at master version , so try hotfix version ,my config folder file as the same to what you say.

**Comment by ShockwaveNN** [comment]
Seems you running convertor like this `./converter `
This is not right, you said it yourself, correct way is like in instruction:
```
LD_LIBRARY_PATH=$PWD/bin NODE_ENV=development-linux NODE_CONFIG_DIR=$PWD/../Common/config ./converter
```

**Comment by stereo720712** [comment]
i success build for hotfix version , i will try master, thanks!

**Comment by ShockwaveNN** [comment]
Can https://github.com/ONLYOFFICE/build_tools/issues/91 be closed too?

Also please not we just released v5.5.3 so we merged hotfix/v5.5.3 branch to master, so they should be same
